perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -192,6 +207,10 @@
 }
 AWS_S3_ENDPOINT_SUFFIX = '.amazonaws.com'
 
+MULTISTREAM_FIXED_DELAY = 5.0
```

Comment:
I added comments and renamed the variables to reflect your comment above. I also changed the delay per stream to 0.1 sec for right now. We'll have to see how that works in experiments to know for sure the right value.